Source/WebCore/accessibility/atspi/AccessibilityObjectAtspi.cpp

```diff
@@ -385,10 +385,10 @@ static unsigned atspiRole(AccessibilityRole role)
     case AccessibilityRole::SystemWide:
     case AccessibilityRole::TableHeaderContainer:
     case AccessibilityRole::ValueIndicator:
-        return Atspi::Role::Unknown;
     case AccessibilityRole::ListMarker:
     case AccessibilityRole::MathElement:
-        RELEASE_ASSERT_NOT_REACHED();
```

Comment:
Looks like a behavior change?

Reply:
Yes, I suppose I should mention that in the commit message.

Reply:
Because I wanted to be lazy and not solve it in a separate commit. I can split it into a second PR if you prefer. Crashing seems like overkill here, and there is already a big list of unknown accessibility roles, so using that seems appropriate.

Reply:
I'll use just this one PR unless you request it be split into two.

Note the title is "fix compiler warnings," not "silence compiler warnings," so behavior changes are probably not completely disallowed.

Reply:
Atspi isn't used by Cocoa ports, right? If so, do your thing.

Reply:
Yes, this is the Linux accessibility technology.

Reply:
Have you seen a crash because of this? At least in some of the cases, like ListMarker, I think it's actually imposible to get that role at this point, because they are specially handled.

Reply:
Nope, I just assumed there was a mistake for some roles to be marked as Unknown and for others to trigger a crash. I'll change it back then, and add a warning comment.

I assume new roles should be marked as Unknown, right?